decoupled-site/backend/Startup.cs

```diff
@@ -122,7 +124,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         app.UseRouting();
 
         app.UseCors(b => b
-            .WithOrigins(new[] { "http://localhost:3000"})
+            .WithOrigins(new[] { $"{_frontendUri}"})
```

Comment:
Please update [doc](https://dash.readme.com/project/digital-experience-platform/v1.4.0-content-graph/docs/on-page-editing-using-content-graph) for this change.